code/ui/generalhud/gametimer/GameTimer.cs

```diff
@@ -37,10 +41,29 @@ public override void Tick()
                     return;
                 }
 
+                lang = ILanguage.GetActiveLanguage();
+
                 bool isWaitingRound = Game.Instance.Round is Rounds.WaitingRound;
 
-                _textLabel.Text = $"{Game.Instance.Round.RoundName}:";
-                _timeLabel.Text = isWaitingRound ? "" : $"{Game.Instance.Round.TimeLeftFormatted}";
+                switch (Game.Instance.Round)
+                {
+                    case Rounds.WaitingRound:
+                        _textLabel.Text = lang.GetTranslation("ROUND_STATE_WAITING");
+                        break;
+
+                    case Rounds.PreRound:
+                        _textLabel.Text = lang.GetTranslation("ROUND_STATE_PREPARING");
+                        break;
+
+                    case Rounds.InProgressRound:
+                        _textLabel.Text = lang.GetTranslation("ROUND_STATE_IN_PROGRESS");
+                        break;
+
+                    case Rounds.PostRound:
+                        _textLabel.Text = lang.GetTranslation("ROUND_STATE_POST_ROUND");
+                        break;
+                }
```

Comment:
That could be handled with RoundName.Upper() concat, but I'm okay with that solution as well as there won't be more round types